src/counts/embezzler.ts

```diff
@@ -0,0 +1,216 @@
+import { canAdventure, getClanLounge, itemAmount } from "kolmafia";
+import {
+  $effect,
+  $familiar,
+  $item,
+  $location,
+  $locations,
+  $monster,
+  $skill,
+  ChateauMantegna,
+  CombatLoversLocket,
+  Counter,
+  CrystalBall,
+  get,
+  have,
+  property,
+  SourceTerminal,
+  sum,
+} from "libram";
+
+const embezzler = $monster`Knob Goblin Embezzler`;
+
+export function embezzlerCount(): number {
```

Comment:
I am hesitant to separate the count from the EmbezzlerFights themselves; I'll accept it as a necessary evil, but it would be nice if we could use some dumb TypeScript bullshit to force us to update them in parallel. It sucks to have to update code in multiple disparate places without an obvious link betwixt.

That being said, this solution is good enough that it can be polished up later

Reply:
I think to do this you'd need to decouple the combat part of the embezzler fight from the count part of the embezzler fight (to not introduce a circular dependency where combat depends on the counts of embezzlers which itself depends on combats) 

maybe we should make 

currently we have 

```ts
export const chainStarters: EmbezzlerFight[] = []; 
export const copySources: EmbezzlerFight[] = []
```
Where each `EmbezzlerFight` knows about `Macro`

Could we instead do something like this?

```ts
export const chainStarters: (macro: MacroClass) => EmbezzlerFight[] = []; 
export const copySources: (macro: MacroClass) => EmbezzlerFight[] = []
```

so as to remove the dependency on Macro and instead have some interface that Macro satisfies?

Reply:
it satisfies the base Macro class, that's for dingdang sure. But I'm not sure if we want to mix those

Reply:
I have to pop out for a few hours.

File renamed.

I'll get to the index files when I get back.

FYI, if it wasn't clear:
IGN: Chartreusenator
Disc: Cinchreusenator